.github/workflows/stale.yml

```diff
@@ -1,28 +1,43 @@
-name: "Stale label"
+---
+# SPDX-license-identifier: Apache-2.0
+##############################################################################
+# Copyright (c) 2023
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Apache License, Version 2.0
+# which accompanies this distribution, and is available at
+# http://www.apache.org/licenses/LICENSE-2.0
+##############################################################################
+
+name: Stale label
+# yamllint disable-line rule:truthy
 on:
   schedule:
     - cron: '0 0 * * *' # Runs daily at 12am UTC
-  branches:
-    - dev-es    
+  push:
+    branches:
+      - dev-es
+
 permissions:
   issues: write
   pull-requests: write
+
 jobs:
   stale:
+    name: Run stale task to label inactive issues
     timeout-minutes: 1 # Default: 360 
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/stale@v7
-      with:
-        repo-token: ${{ secrets.GITHUB_TOKEN }}
-        stale-issue-message: 'This issue/PR is stale because it has been inactive for 60 days. This issue/PR will be closed automatically with additional 20 days of inactivity. If an update/comment occur on stale issue/PR, the stale label will be removed and the timer will restart. Closed issue/PR can be reopened if necessary.'
-        stale-pr-message: 'This issue/PR is stale because it has been inactive for 60 days. This issue/PR will be closed automatically with additional 20 days of inactivity. If an update/comment occur on stale issue/PR, the stale label will be removed and the timer will restart. Closed issue/PR can be reopened if necessary.'
-        exempt-issue-labels: 'maintainers, lang/es'
-        exempt-pr-labels: 'maintainers'
-        stale-pr-label: 'stale'
-        stale-issue-label: 'stale'
-        days-before-stale: 60
-        days-before-close: 20
-        only-labels: 'lang/es'
-        close-issue-label: 'close/stale'
-        close-pr-label: 'close/stale' # Add the label for mainteiners track the close pr labels.
\ No newline at end of file
+      - uses: actions/stale@v8.0.0
```

Comment:
it was tested with this new version this action?

Reply:
I tried to test this locally with `act`  tool but requires some env vars. Besides using the latest version released, this semver format is validated by dependabot

Reply:
ok!! let's merge it!